tests/NativeCalculatorTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Brick\Math\Tests;
```

Comment:
Should be Brick\Math\Tests\Internal\Calculator and file moved to the appropriate directory